packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -126,16 +136,39 @@ const defaultMuiThemeOptions: ThemeOptions = {
             )} 0 0 0 0.1rem`,
             borderColor: theme.palette.primary.main,
           },
+          '& fieldset': {
+            border: 'none',
+          },
         }),
         input: ({ theme }) => ({
           padding: '14px 12px',
         }),
+        multiline: {
+          padding: '0px',
+        },
       },
     },
     MuiInputLabel: {
       defaultProps: {
         shrink: true,
       },
+      styleOverrides: {
+        root: ({ theme }) => ({
+          fontWeight: 700,
+          mb: '4px',
+          fontSize: '14px',
+          transform: 'none',
+        }),
+      },
+    },
+    MuiTextField: {
+      styleOverrides: {
+        root: ({ theme, ownerState }) => ({
+          [`& .${inputBaseClasses.root}`]: {
+            marginTop: ownerState.label && theme.spacing(3),
+          },
+        }),
```

Comment:
Only add margin for the label when the TextField actually has a label, so that TextFields without labels don't have extra spacing added (e.g. CopyToClipboardInput)